src/test/scala/CommitCursorBadResponseSpec.scala

```diff
@@ -89,7 +89,10 @@ class CommitCursorBadResponseSpec(implicit ec: ExecutionEnv) extends Specificati
         nakadiSource.source
           .map { subscriptionEvent =>
             subscriptionsClient
-              .commitCursors(subscriptionId, SubscriptionCursor(List(subscriptionEvent.cursor)), nakadiSource.streamId)
+              .commitCursors(subscriptionId,
+                             SubscriptionCursor(List(subscriptionEvent.cursor)),
+                             nakadiSource.streamId,
+                             eventBatch = false)
```

Comment:
Can you add a separate test case where `eventBatch = true` so that we can test the original scenario as well?

Reply:
Added one for the “keep-alive” case and reverted the one I changed before.